### ISSUE TITLE
Production-area page

### ISSUE THREAD

**Issue by oliverroick** [earth-genome/foodtwin-global-app#20]
**Context**

This ticket is to set up the production-area page, including the map and a barebones scrollable sidebar with the empty panels. We'll fill in the content for the panels in separate tickets. 

**Acceptance criteria**

- The sidebar contains five panels:
  - Overview
  - Crops produced
  - Shipping nodes
  - Shipping routes
  - Impact on people
- The sidebar is scrollable, the panel closest to the top is considered the "active" panel. 
- Upon panel activation the map view is updated as follows:
  - Overview: View is on the production area
  - Crops produced: View is on the production area
  - Shipping nodes: View shows all shipping nodes connected to the production area, and the routes connecting the production area with the shipping nodes.
  - Shipping routes: View shows all shipping routes directly connected to the production area, or indirectly connected through shipping nodes. 
  - Impact on people: View is on the areas where people would be impacted from a disruption.

**Comment by wrynearson** [comment]
@faustoperez @vgeorge can this be closed?

**Comment by vgeorge** [comment]
@wrynearson yes, I believe we have dedicated tickets for the remaining work.
